mgmtsystem_action/mgmtsystem_action.py

```diff
@@ -80,13 +80,8 @@ def case_open(self):
 
         for case in self:
             values = {'active': True}
-
-            stages = self.env['mgmtsystem.action.stage']
-            values['stage_id'] = stages.search([
-                ['is_ending', '=', False],
-                ['is_starting', '=', False]
-            ]).id
-
+            values['stage_id'] = self.env.ref(
+                'mgmtsystem_action.stage_open').id
```

Comment:
@dreispt I don't have a lot of time to spend on this issue and backporting could affect other people who use the module. What about adding limit=1 and order='sequence' ?